.gitignore

```diff
@@ -16,3 +16,5 @@ output/
 bower_components
 node_modules
 content/extensions
+.venv
+package-lock.json
```

Comment:
I don't think package-lock.json is supposed to go in .gitignore.

Reply:
No, it should go into version control.